es6-promises/main.js

```diff
@@ -0,0 +1,7 @@
+import takeAChance from './take-a-chance.js';
+
+const obj = takeAChance('Kevin');
+
+const error = new Error('Failure...');
+
+obj.then(() => console.log('Success!')).catch(() => console.error(error));
```

Comment:
This isn't quite right. The `then` arrow function receives the resolved value as a parameter while the `catch` arrow function receives the rejected value. Those are what you want to log.

Something like:
`obj.then((value) => console.log(value)).....`

Also, you can just chain onto `takeAChance` instead of using a local variable.